packages/quick_actions/quick_actions_android/android/src/main/java/io/flutter/plugins/quickactions/QuickActionsPlugin.java

```diff
@@ -74,6 +74,7 @@ public boolean onNewIntent(Intent intent) {
     }
     // Notify the Dart side if the launch intent has the intent extra relevant to quick actions.
     if (intent.hasExtra(MethodCallHandlerImpl.EXTRA_ACTION) && channel != null) {
+      channel.invokeMethod("getLaunchAction", null);
```

Comment:
Why are calling Java code from Java code via the method channel? And why are you calling a getter but not using the result?

Reply:
I want to reuse the logic in `getLaunchAction`. 

Since this seems to be a bad idea, I have tried another way that is exposing a public method from `MethodCallHandlerImpl`. Do let me know if this is okay.